.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: python
+python:
+    - "2.7"
+    - "2.6"
+before_install:
+    - git config --global user.email "OpenStack_TravisCI@f5.com"
+    - git config --global user.name "Travis F5 Openstack"
+install:
+    - pip install requests hacking pytest pytest-cov
+script:
+    - flake8 .
+    - py.test --cov --ignore=test/
+notifications:
+    slack:
+      rooms:
+        - f5openstackdev:$SLACK_PROJECT_TOKEN
```

Comment:
In the properties page for the repo on Travis there is a Settings tab that you can create environment variables.  Be sure when you add them to make them hidden (which I think is default) so that they don't show up in the travis-ci logs.